blog/2024-02-01-py-client-v4-release/index.mdx

```diff
@@ -0,0 +1,436 @@
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
+import ManageCollectionsCode from '!!raw-loader!/_includes/code/howto/manage-data.collections.py';
+
+![Weaviate Python Client v4 Release](./img/hero.png)
+
+The `v4` release of the Weaviate Python client library has landed, with a brand new API that is more powerful and easier to use 🎉🥳. This is a major update, as reflected by the bump in the major version number.
+
+It still includes the existing (`v3`) API for now, so you can continue to use it without any changes. But we do note and the `v3` API is considered deprecated, and will be removed in the future (estimated ~6 months).
+
+If you're keen to get on with it, just `pip` install it with:
+
+```bash
+pip install -U weaviate-client
+```
+
+But if you're curious about what's new, read on - there's a lot that we are excited about!
+
+## Key ideas
+
+The `v4` client does things a little differently than the `v3` client. Two of the key ideas where it differs are:
+
+- Collection-based interactions, and
+- Default property returns to queries.
+
+Take this query, for example:
+
+<FilteredTextBlock
+    text={PythonCode}
+    startMarker="# START HybridQueryExample"
+    endMarker="# END HybridQueryExample"
+    language="py"
+/>
+
+Here, you'll notice that the query:
+- is performed on a collection (`questions`), not the `client` itself, and
+- does not specify any properties to return.
+
+This is because the `v4` client is collection-based, and will return all properties (except `blob`s and references) by default. We think this makes the code more concise, and easier to use.
+
+### IDE assistance / autocomplete
+
+And writing the queries is now also much easier, with more IDE assistance and autocomplete. For example, here's a query that uses the `Filter` class:
+
+![Weaviate Python Client v4 Release](./img/pyv4_ide_example2_filter.gif)
+
+Notice all the nice autocomplete suggestions as we type! This extends to a great majority of operations, from collection creation to object insertion to query building.
+
+### Speed improvements
+
+The new client is much faster than the `v3` client, too. This is because it uses gRPC extensively under-the-hood.
+
+The speed improvements during imports will increase along with the request size. As some rough examples, a 50s import of `ANN` benchmark dataset with the `v3` client will take ~30s with the `v4` client,
+
+Queries will also be faster with the `v4` client.
+
+Anecdotally, a query on an HNSW index with an `ef` of 16 that had a QPS (queries per second) of 162 with the `v3` client had a QPS of 1560 with the `v4` client. Another query with an `ef` of 256 had a QPS of 110 with the `v3` client, and a QPS of 193 with the `v4` client. (This is a function of how long each query takes server-side, as that is not affected by the client improvements.)
```

Comment:
I think these are too many numbers, how about...

Queries will also be faster with the `v4` client and we see improvements from ~40% up to 80%."

and mention that we will provide an in-detail analysis in a following blogpost

Reply:
Fair